### PR TITLE
[easy] fix graph test  + error message

### PR DESCRIPTION
**Pull request by dpeng817**
I think this test was previously weird; we should be asserting on the expected error message, not the _lack_ of an error message. 

Also, fix the error message itself, the argument doesn't need to be a "keyword" argument. Just an argument.

### CODE REVIEW

**Comment by dpeng817** [comment]
* **#26352** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26352?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26352?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: python_modules/dagster/dagster/_core/definitions/asset_dep.py [hunk 1]
```diff
@@ -28,6 +28,7 @@ class AssetDep(
         [
             ("asset_key", PublicAttr[AssetKey]),
             ("partition_mapping", PublicAttr[Optional[PartitionMapping]]),
+            ("io_manager_key", PublicAttr[Optional[str]]),
```
Comment (jamiedemaria) [inline python_modules/dagster/dagster/_core/definitions/asset_dep.py:31]: is this meant to be part of this pr? seems unrelated to fixing the test

**Review by jamiedemaria — DISMISSED** [review]

File: python_modules/dagster/dagster/_core/definitions/asset_dep.py (hunk 1 above)
Comment (dpeng817) [reply python_modules/dagster/dagster/_core/definitions/asset_dep.py]: whoops. removed.

**Review by jamiedemaria — APPROVED** [review]
lgtm!
